.github/workflows/test-build.yml

```diff
@@ -0,0 +1,41 @@
+name: ci
```

Comment:
[nit] Could be capitalised, this is meant to be a human-friendly name used in the UI